aws-ami.pkr.hcl

```diff
@@ -89,6 +138,7 @@ build {
       "GRAPHDB_VERSION=${var.gdb_version}",
     ]
     # Execute the GraphDB installation script.
-    inline = ["sudo -E bash /tmp/install_graphdb.sh"]
+    inline      = ["sudo -E bash /tmp/install_graphdb.sh"]
+    max_retries = 3
```

Comment:
I didn't catch this previously, what's the reason for adding it? Not that it's bad but let's see what breaks the build and try to avoid it by fixing it.

Reply:
I've added this due to the Ubuntu issue not finding its packages (`apt-get -o DPkg::Lock::Timeout=300 install -y unzip jq nvme-cli openjdk-11-jdk bash-completion`) 50% of the time. Instead of cleaning the apt cache and then installing this seems to do the trick.